internal/storage/v1/elasticsearch/spanstore/reader.go

```diff
@@ -223,34 +222,29 @@ func timeRangeIndices(indexName, indexDateLayout string, startTime time.Time, en
 }
 
 // GetTrace takes a traceID and returns a Trace associated with that traceID
-func (s *SpanReader) GetTrace(ctx context.Context, query spanstore.GetTraceParameters) (*model.Trace, error) {
+func (s *SpanReader) GetTrace(ctx context.Context, traceIDs map[dbmodel.TraceID]string) (map[dbmodel.TraceID][]*dbmodel.Span, error) {
 	ctx, span := s.tracer.Start(ctx, "GetTrace")
 	defer span.End()
 	currentTime := time.Now()
 	// TODO: use start time & end time in "query" struct
-	traces, err := s.multiRead(ctx, []model.TraceID{query.TraceID}, currentTime.Add(-s.maxSpanAge), currentTime)
+	traces, err := s.MultiRead(ctx, traceIDs, currentTime.Add(-s.maxSpanAge), currentTime)
 	if err != nil {
 		return nil, es.DetailedError(err)
 	}
 	if len(traces) == 0 {
 		return nil, spanstore.ErrTraceNotFound
 	}
-	return traces[0], nil
+	return traces, nil
 }
 
-func (s *SpanReader) collectSpans(esSpansRaw []*elastic.SearchHit) ([]*model.Span, error) {
-	spans := make([]*model.Span, len(esSpansRaw))
-
+func (s *SpanReader) collectJsonSpans(esSpansRaw []*elastic.SearchHit) ([]*dbmodel.Span, error) {
```

Comment:
```suggestion
func (s *SpanReader) collectSpans(esSpansRaw []*elastic.SearchHit) ([]*dbmodel.Span, error) {
```
unnecessary renaming